example_site/src/components/ConfigurationControls.js

```diff
@@ -192,25 +192,61 @@ export default function ConfigurationControls({ isOpen }) {
           this gets your gears turning about how the
           Waymark SDK could work for you."
         />
-
         <h4>To get started, choose an example partner</h4>
+      </div>
 
-        <select
-          name="partner"
-          ref={register({ required: true })}
-          onChange={(e) => setPartner(e.target.value)}
-          defaultValue={partnerPresets.default}
-        >
-          <option value={partnerPresets.default}>Default Partner</option>
-          <option value={partnerPresets.spectrum}>Spectrum Reach</option>
-          <option value={partnerPresets.custom}>Custom</option>
-        </select>
-        
-        <button 
-          className="submit-button configuration-submit-button">
-          See How It Works
-        </button>
+      <div className='three-columns'>
+        <div className='configuration-controls-subsection'>
+          <button
+            className='partner-button'
+            onClick={() => setPartner(partnerPresets.generic)}
+          >
+            <img 
+              style={{
+                  width: '200px',
+                  height: '200px',
+                  objectFit: 'contain',
+                }}
+              src="https://socialproof-prod.imgix.net/video_creatives/videotemplatevariant/thumbnail/1209_1597265598.png?ixlib=react-8.6.4&auto=compress%2Cformat&fit=max&w=512"
```

Comment:
I know these are all the same for the time being, but what do you think about adding this to the `partnerConfigurations` object and then just looping through? Remember that this will make it easier for us to add additional cards in the future. 

This will require altering the structure of `partnerConfigurations` to include a `configuration` object and a place to store the thumbnail image.